### ISSUE TITLE
I found the screenshot is black

### ISSUE THREAD

**Issue by Bekeys** [fluttercandies/ff_native_screenshot#1]

**Comment by Bekeys** [comment]
And the screenshot is data null outside the application

**Comment by Bekeys** [comment]
@CaiJingLong @AlexV525 @cr1992 @zmtzawqlp

**Comment by zmtzawqlp** [comment]
i can't get you

**Comment by amigax** [comment]
Yes it is black, what a nightmare, theres no way to take a native screenshot now on later androids.

**Comment by n1kStas** [comment]
did you manage to fix it?

UPD: I offer you my screenshot implementation https://pub.dev/packages/screenshot_ntv

**Comment by PierreKasparian** [comment]
> did you manage to fix it?
> 
> UPD: I offer you my screenshot implementation https://pub.dev/packages/screenshot_ntv
@n1kStas 
Does this package allows to take screenshot when the app is running in background ?
For example i want to do a dash bubble that displays over other apps, and when we press it, it takes a screenshot.
Is this possible with your package ?
Thank you

**Comment by thisames** [comment]
> você conseguiu consertar?
> 
> UPD: ofereço minha implementação de captura de tela https://pub.dev/packages/screenshot_ntv

your solution actually works great! congratulations and thank you. all my solutions hit the black screen.
